sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryTracer.java

```diff
@@ -189,14 +193,50 @@ public Context getSharedSpanBuilder(String spanName, Context context) {
         return context.addData(SPAN_BUILDER_KEY, getSpanBuilder(spanName, context));
     }
 
+    @Override
+    public void addEvent(String eventName, Map<String, Object> attributes, Instant timestamp) {
```

Comment:
We generally take `OffsetDateTime` for timestamps.

Reply:
Updated it to OffsetDateTime input.